src/main/java/com/onlinebookstore/model/User.java

```diff
@@ -27,4 +36,40 @@ public class User {
     private String lastName;
     @Column(name = "shipping_address")
     private String shippingAddress;
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(name = "users_roles",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id")
+    )
+    private Set<Role> roles = new HashSet<>();
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return roles;
+    }
+
+    @Override
+    public String getUsername() {
+        return email;
+    }
+
+    @Override
+    public boolean isAccountNonExpired() {
+        return true;
+    }
+
+    @Override
+    public boolean isAccountNonLocked() {
+        return true;
+    }
+
+    @Override
+    public boolean isCredentialsNonExpired() {
+        return true;
+    }
+
+    @Override
+    public boolean isEnabled() {
+        return true;
+    }
```

Comment:
Think about how we should add columns like is_deleted. Should we add them directly to existing scripts or there is another better way?
Should we add soft-delete here?

Reply:
It wasn't mentioned in the task...

Reply:
Should we?

Reply:
It is not in the task descriprion, but I think it worth to add

Reply:
You can do it in next hw